src/main/java/com/epam/preprod/sirenko/Demo.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.containers.CommandContainer;
+import com.epam.preprod.sirenko.dao.ProductDAO;
+
+import java.io.*;
+
+public class Demo {
+	CommandContainer commandContainer;
+	ProductDAO productDAO;
+	
+	public Demo() {
```

Comment:
We dont need constructor here because we dont need to create instance of Demo

Reply:
deleted